modules/localStore.js

```diff
@@ -0,0 +1,24 @@
+class LocalStore {
+  static getBooks() {
+    let books;
+    const localBook = localStorage.getItem('local');
+    if (!localBook) {
+      books = [];
+    } else {
+      books = JSON.parse(localBook);
+    }
+    return books;
+  }
+
+  static addBooks(newBook) {
```

Comment:
- [ ] Kindly make use of ES6 arrow function for your approach to be of latest and it would help you in React module.

Reply:
Please @Olamarx  I didn't succeed to change the static functions to arrow functions. The last I checked, it's not possible to do that in classes. 😁 Please could you help me to achieve that? Thanks in advance.